front/tests/utils/index.ts

```diff
@@ -188,3 +189,26 @@ export async function performOnSpecificOSAndBrowser(
     console.info(skipMessage);
   }
 }
+
+/**
+ * Utility function to get translations based on the project language.
+ *
+ * @param {string} projectLanguage - The language of the project
+ * @param {{ en: T; fr: T }} translations - An object containing translations for English and French.
+ * @returns {T} - The translation object corresponding to the specified project language.
+ * @throws {Error} - Throws an error if the project language is unsupported.
+ */
+
+export function getTranslations<T = unknown>(
+  projectLanguage: string,
+  translations: { en: T; fr: T }
+): T {
```

Comment:
- why not get the language from the process.env here ?
- you don't need to precise unknown
```suggestions
export function getTranslations<T>(translations: { en: T; fr: T }): T {
  const projectLanguage = process.env.PROJECT_LANGUAGE;
```

Reply:
Done